src/components/Projects/ProjectBlock.tsx

```diff
@@ -0,0 +1,127 @@
+import { BodyText, HeaderText, PrimaryAccentText } from 'components/Text'
+import { JSX } from 'preact/jsx-runtime'
+import { useRef } from 'preact/hooks'
+import Button from 'components/Button'
+import Tilt from 'react-parallax-tilt'
+import classnames, {
+  alignItems,
+  display,
+  dropShadow,
+  flexDirection,
+  gap,
+  inset,
+  justifyContent,
+  margin,
+  position,
+  rotate,
+  scale,
+  space,
+  width,
+  zIndex,
+} from 'classnames/tailwind'
+
+type Element = JSX.Element
+
+const wrapper = (reverse?: boolean, centered?: boolean) =>
+  classnames(
+    position('relative'),
+    display('flex'),
+    gap('gap-8'),
+    flexDirection(
+      { 'lg:flex-row-reverse': reverse, 'flex-col-reverse': reverse },
+      'flex-col',
+      'lg:flex-row'
+    ),
+    alignItems('items-center'),
+    justifyContent(centered ? 'justify-center' : 'justify-evenly'),
+    width('w-auto', 'md:w-full'),
+    margin('mx-4'),
+    zIndex('z-10')
+  )
+
+const descriptionBlock = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  width('w-fit', 'sm:w-96'),
+  alignItems('items-start')
+)
+const cardHeader = space('space-y-4')
+const cardBody = space('space-y-6')
```

Comment:
Maybe we can move `space` into `<div className={...}>`. But it's up to you

Reply:
We have all classnames in consts, let's leave it like this